tc/core/halide2isl.cc

```diff
@@ -402,12 +402,12 @@ isl::schedule makeScheduleTreeHelper(
     // this loop var at the same index.
     isl::multi_union_pw_aff mupa;
     body.get_domain().foreach_set([&](isl::set s) {
-      isl::aff loopVar(
+      isl::aff newLoopVar(
```

Comment:
I wonder if it is not better solved by only capturing what we need in the lambda.

Reply:
@ftynse , this won't help, I still get a compilation error if it shadows a name

Reply:
That's a badly implemented warning then :(

Can we at least have something like `loopVariable` instead of `newLoopVar` here?  Otherwise it's misleading in loop creation context....